src/blas/backends/mklcpu/cpu_batch.cpp

```diff
@@ -20,6 +20,7 @@
 #include <CL/sycl.hpp>
 
 #include "cpu_common.hpp"
+#include "onemkl/blas/detail/mklcpu/onemkl_blas_mklcpu.hpp"
```

Comment:
Why is this header file needed here now?

Reply:
Because in this header we have functions definitions with ONEMKL_EXPORT flag, which will help us to export them for windows

Reply:
So, it is only required for Windows, right?

Reply:
At this moment, yes, but it doesn't hurt having it on Linux. I suggest to leave it as is.

Reply:
Also it's nice to hide Linux internal symbols, so these changes can help us to make it using the same macro for Linux

Reply:
Thanks for the explanation.